koku/api/settings/tags/serializers.py

```diff
@@ -20,6 +20,12 @@ class SettingsTagSerializer(serializers.Serializer):
     class Meta:
         model = EnabledTagKeys
 
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        add_source_type = {"source_type": instance.provider_type, **representation}
+        add_source_type.pop("provider_type")
+        return add_source_type
```

Comment:
This is a nitpick, but `to_representation` acts on the entire instance, not just the source-type. I would use something more generic than the `add_source_type` variable name, such as `data`:
```suggestion
    def to_representation(self, instance):
        data = super().to_representation(instance)
        data["source_type"] = data.pop("provider_type")
        return data
```